Assets/Scripts/PlayerController.cs

```diff
@@ -7,7 +7,27 @@
 public class PlayerController : MonoBehaviour {
 
     [SerializeField]
-    private float playerAcceleration, maxSpeed, jumpForceForward, jumpForceUp, bulletSpeed, recoilSpeed, overheatAdd, overheatSubtract;
+    [Tooltip("Max Speed that the Player can Move")]
+    private float maxSpeed;
+    [SerializeField]
+    [Tooltip("How far Forward the Player Jumps on Jump")]
```

Comment:
If it's a force your'e added, it's not measured in distance, but your tooltip suggests it's a distance. More accurate description would be "how much forward force to apply  when the player jumps". or something of that nature.